modules/virt-nmstate-example-policy-configurations.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-nmstate-example-policy-configurations"]
```

Comment:
```suggestion
[id="virt-nmstate-example-policy-configurations_{context}"]
```
Each module should have a module anchor with {context} that must be lowercase and must match the module’s file name. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata.

Reply:
Correct. My apologies. My own fault for copying and pasting certain file metadata.

Reply:
:heavy_check_mark: